xml/System.Runtime/AmbiguousImplementationException.xml

```diff
@@ -73,9 +73,9 @@
         <Parameter Name="innerException" Type="System.Exception" />
       </Parameters>
       <Docs>
-        <param name="message">To be added.</param>
-        <param name="innerException">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="message">The localized error message string.</param>
+        <param name="innerException">The exception that is the cause of the current exception. If the <paramref name="innerException" /> parameter is not a null reference (<see langword="Nothing" /> in Visual Basic), the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</param>
```

Comment:
@jkotas Is the deferred catch here true?  MissingMethodException also says it, but I'm not familiar with an inner exception being a candidate for a catch, and a thrown InvalidOperationException inside a manually thrown MissingMethodException certainly didn't trigger `catch (InvalidOperationException)` in a test just now.

Seems like this should just be

```suggestion
        <param name="innerException">The exception that is the cause of the current exception.</param>
```